lib/bank_account/account.rb

```diff
@@ -0,0 +1,39 @@
+module BankAccount
+  class Account
+
+    attr_accessor :id, :balance
+
+    #Account needs to be opened with a positive balance. It will throw an error message if you #attempt to open with a negative balance.
+    def initialize (id, initial_balance)
+      if initial_balance < 0
+        raise ArgumentError, 'You cannot use a negative number for your initial balance'
+      end
+      @id = id
+      @balance = initial_balance
+    end
+
+
+    def withdraw(amount)
+      if @balance - amount < 0
+        print "This transaction cannot be completed because you don't have enough money."
```

Comment:
Instead of `print`, if you use `warn`, it'll let readers know this is a potential problem